src/components/PopupWithForm.js

```diff
@@ -2,10 +2,22 @@ import Popup from "./Popup.js";
 
 export default class PopupWithForm extends Popup {
   constructor(popupSelector, handleFormSubmit) {
-    super( popupSelector );
+    super(popupSelector);
     this._handleFormSubmit = handleFormSubmit;
     this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._submitButton = this._popupForm.querySelector(".modal__button");
+    this._deleteCardButton = this._popupElement.querySelector("#modal__card-delete-button");

```

Comment:
`    this._deleteCardButton = this._popupElement.querySelector("#modal__card-delete-button");
`

This is not used in `PopupWithForm` (and you can't use it anyway)